ramls/routing-list.raml

```diff
@@ -0,0 +1,81 @@
+#%RAML 1.0
+title: "mod-orders-storage"
+baseUri: http://github.com/folio-org/mod-orders-storage
+version: v1.0
+
+documentation:
+  - title: Routing List
+    content: <b>This module implements the CRUD interface. This API is intended for internal use only. </b>
+
+types:
+    routing-list: !include acq-models/mod-orders-storage/schemas/routing_list.json
+    routing-list-collection: !include acq-models/mod-orders-storage/schemas/routing_list_collection.json
+    errors: !include raml-util/schemas/errors.schema
+    UUID:
+     type: string
+     pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+    orderable: !include raml-util/traits/orderable.raml
+    pageable: !include raml-util/traits/pageable.raml
+    searchable: !include raml-util/traits/searchable.raml
+    validate: !include raml-util/traits/validation.raml
+
+resourceTypes:
+    collection: !include raml-util/rtypes/collection.raml
+    collection-item: !include raml-util/rtypes/item-collection.raml
+
+/orders-storage/routing-lists:
+  type:
+    collection:
+      exampleCollection: !include acq-models/mod-orders-storage/examples/routing_list_collection.sample
+      exampleItem: !include acq-models/mod-orders-storage/examples/routing_list_get.sample
+      schemaCollection: routing-list-collection
+      schemaItem: routing-list
+  get:
+    description: Get collection of routing lists
+    is: [
+      searchable: { description: "CQL query", example: "name=MyRoutingList" },
+      pageable
+    ]
+  post:
+    description: Create a new routing list record
+    body:
+      application/json:
+        type: routing-list
+        example:
+          strict: false
+          value: !include acq-models/mod-orders-storage/examples/routing_list_get.sample
+    responses:
+      201:
+        description: "Returns a newly created item, with server-controlled fields like 'id' populated"
+        body:
+          application/json:
+            example: !include acq-models/mod-orders-storage/examples/routing_list_get.sample
+      400:
+        description: "Bad request, e.g. malformed request body or query parameter. Details of the error (e.g. name of the parameter or line/character number with malformed data) provided in the response."
+        body:
+          text/plain:
+            example: "Unable to add routing list -- malformed JSON at 13:3"
+      401:
+        description: "Not authorized to perform requested action"
+        body:
+          text/plain:
+            example: "Unable to add routing-list -- unauthorized"
+      500:
+        description: "Internal server error, e.g. due to misconfiguration"
+        body:
+          text/plain:
+            example: "Internal server error, contact administrator"
```

Comment:
for new APIs we should not provide text/plain response, we should provide application/json if some error is occurs, use the same declaration as for other APIs without text/plain